packages/components/.storybook/main.js

```diff
@@ -0,0 +1,78 @@
+const path = require('path');
```

Comment:
we don't need `addons.js`, `config.js`, `sass-loader` and `webpack` in the new version of storybook